code/game/objects/items/weapons/teleportation.dm

```diff
@@ -172,18 +175,45 @@ Frequency:
 
 	var/t1 = input(user, "Please select a teleporter to lock in on.", "Hand Teleporter") in L
 
-	if ((user.get_active_hand() != src || user.stat || user.restrained()))
+	if((user.get_active_hand() != src || user.stat || user.restrained()))
 		return
-	if(portals.len >= 3)
+	if(charges < 1)
 		user.show_message("<span class='notice'>\The [src] is recharging!</span>")
 		return
 	var/T = L[t1]
-	for(var/mob/O in hearers(user, null))
-		O.show_message("<span class='notice'>Locked In.</span>", 2)
-	var/obj/effect/portal/P = new /obj/effect/portal( get_turf(src) )
-	P.target = T
-	P.creator = src
-	portals += P
+
+	if(prob(5))

```

Comment:
Why are you dumping everyone on zlevel three though. You know zlevel 3 on some maps could be centcomm instead, right?